test/functional/kaui/transactions_controller_test.rb

```diff
@@ -49,6 +65,23 @@ class Kaui::TransactionsControllerTest < Kaui::FunctionalTestHelper
     end
   end
 
+  test 'should fix transaction state' do
+    parameters = {
+      :account_id => @account.account_id,
+      :transaction => {
+        :payment_id => @payment.payment_id,
+        :transaction_id => @payment.transactions[0].transaction_id,
+        :status => 'PENDING'
```

Comment:
Similar to my remark above regarding the bundle, this could have side effects (the payment isn't successful --
 yet -- anymore, meaning any associated invoice isn't paid -- yet -- for instance). Better to generate new data for this.